app/assets/stylesheets/miq_tree.scss

```diff
@@ -17,3 +17,9 @@
     }
   }
 }
+
+.react-tree-view {
+    .opacity {
+        opacity: 0.4;
```

Comment:
This was already here, line 12 ... which probably means the new tree does not use `list-group-item` for the tree items?

But if so, then we have the exact same problem for the rest of this file..

=> `.strikethrough`, `.red` and `.no-cursor` should get a `.react-tree-view` copy too?

(Or change the tree to use the same class?)

Reply:
Well, `no-cursor` and `red` might be a valid issue, but AFAIK `strikethrough` is only being used in the left-side trees. Anyway, we should probably keep both implementations symmetric and in this same file.